src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -1273,13 +1274,20 @@ def ramp_fit_overall(
     # Suppress, then re-enable harmless arithmetic warnings
     warnings.filterwarnings("ignore", ".*invalid value.*", RuntimeWarning)
     warnings.filterwarnings("ignore", ".*divide by zero.*", RuntimeWarning)
+
     slope_int = the_num / the_den
+
+    # Adjust DQ flags for NaNs.
+    wh_nans = np.where(np.isnan(slope_int))
```

Comment:
Minor query: is the `np.where` even needed?  My understanding is that the following lines would work without it.